src/misc.jl

```diff
@@ -4,10 +4,11 @@
 #
 # See: https://en.wikipedia.org/wiki/Multivariate_gamma_function
 #
-function logmvgamma(p::Int, a::Float64)
-    res::Float64 = p * (p - 1.0) / 4.0 * log(pi)
+function logmvgamma(p::Int, a::Real)
+    # NOTE: one(a) factors are here for type stability for, e.g., Float32
+    res = p * (p - 1) * log(pi * one(a)) / 4
```

Comment:
```jl
julia> oftype(Int, pi)
WARNING: oftype{T}(::Type{T},c) is deprecated, use convert(T,c) instead.
 [inlined code] from ./error.jl:26
 in depwarn(::ASCIIString, ::Symbol) at ./deprecated.jl:64
 in oftype(::Type{Int64}, ::Irrational{:π}) at ./deprecated.jl:50
 in eval(::Module, ::Any) at /Users/andreasnoack/julia-dev/usr/lib/julia/sys.dylib:-1
while loading no file, in expression starting on line 0
ERROR: MethodError: Cannot `convert` an object of type Irrational{:π} to an object of type Int64
This may have arisen from a call to the constructor Int64(...),
since type constructors fall back to convert methods.
Closest candidates are:
  convert(::Type{Int64}, ::Int8)
  convert(::Type{Int64}, ::Int16)
  convert(::Type{Int64}, ::Int32)
  ...
 in oftype(::Type{Int64}, ::Irrational{:π}) at ./deprecated.jl:52
 in eval(::Module, ::Any) at /Users/andreasnoack/julia-dev/usr/lib/julia/sys.dylib:-1

julia> one(Int)*pi
3.141592653589793
```